keyboards/1upkeyboards/super16/keymaps/nblyumberg/config.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+// Time out one shot layers after 3 seconds
+#define ONESHOT_TIMEOUT 3000
+
+#define RGBLED_NUM 16
+#define RGBLIGHT_LAYERS //Enable Layer indicators with RGB
```

Comment:
Given you no longer have the layer stuff,
```suggestion
```

Reply:
Removed and commited again